src/sentry/static/sentry/app/stores/projectsStore.jsx

```diff
@@ -23,6 +24,28 @@ const ProjectsStore = Reflux.createStore({
     this.trigger(new Set(Object.keys(this.itemsById)));
   },
 
+  onChangeSlug(prevSlug, newSlug) {
+    let prevProject = this.getBySlug(prevSlug);
```

Comment:
`const`.

Reply:
There's no value in using `const` right now when most of the codebase is using `let`

Reply:
Yeah but we won't get to a const codebase without moving away from let :)

Reply:
It's going to be confusing (esp in reviews) since existing code uses `let`. It's better to bulk update to `const`.